test/controllers/v1/bundles_controller_test.exs

```diff
@@ -64,24 +64,33 @@ defmodule Cog.V1.BundlesControllerTest do
     end
   end)
 
+  test "rejects an unsupported bundle version", %{authed: requestor} do
+    config_path = make_config_file("unsupported_config.yaml", contents: unsupported_config)
+    upload = %Plug.Upload{path: config_path, filename: "unsupported_config.yaml"}
+
+    conn = api_request(requestor, :post, "/v1/bundles", body: %{bundle: %{config_file: upload}}, content_type: :multipart)
+
+    %{"errors" => errors} = Poison.decode!(conn.resp_body)
+
+    assert errors == ["Invalid config file.",
+                      "Error near #/cog_bundle_version: cog_bundle_version 2 is not supported. Please update your bundle config to version 4."]
+  end
+
   test "accepts an upgradable config", %{authed: requestor} do
-    config_path = make_config_file("old_config.yaml", contents: old_config)
-    upload = %Plug.Upload{path: config_path, filename: "old_config.yaml"}
+    config_path = make_config_file("upgradable_config.yaml", contents: upgradable_config)
```

Comment:
This whole test might just go away instead... the idea of "upgradable configs" was from when we'd make automatic changes to go from v2 to v3. We don't actually have that code anymore.